assets/src/scss/layout/_navbar.scss

```diff
@@ -117,8 +115,27 @@ $navbar-default-height: 60px;
   }
 
   @include small-and-less {
+    width: 100vw;
+    inset-inline-start: -100vw;
+
+    @supports not (inset-inline-start: -100vw) {
```

Comment:
You can probably use a single `@supports` rule testing for `insert-inline-start: -100vw`. I don't think there's cases where a browser would support only 1 of the 2.

Reply:
Just make sure that `@supports` is supported by the browser in question :grin:

Otherwise the code should be reverse to avoid using `not`.

Reply:
I've added `not` since `insert-inline-start` is available in most of the browsers but there some few that aren't. Doing this we'll avoid to add extra code in most cases.

Does it make sense?

Reply:
Yes. But I meant that if that browser doesn’t supports `@supports` either then this won’t work.

I see it’s [supported](https://caniuse.com/css-featurequeries) since Safari 9, so we are good 👍🏻